pkg/fakerp/server.go

```diff
@@ -19,11 +19,13 @@ import (
 	"github.com/ghodss/yaml"
 	"github.com/sirupsen/logrus"
 
-	"github.com/openshift/openshift-azure/pkg/api"
-	v20180930preview "github.com/openshift/openshift-azure/pkg/api/2018-09-30-preview/api"
+	internalapi "github.com/openshift/openshift-azure/pkg/api"
+	externalapi "github.com/openshift/openshift-azure/pkg/api/2018-09-30-preview/api"
```

Comment:
Revert this change, I am adding support for the admin api which is also external.